src/main/java/com/chessmates/repository/MockGameRepository.groovy

```diff
@@ -13,22 +11,22 @@ import org.springframework.stereotype.Repository
 class MockGameRepository implements GameRepository {
     Logger logger = LoggerFactory.getLogger(MockGameRepository)
 
-    Map<String, Game> store = new HashMap()
+    Map store = new HashMap()
 
     @Override
-    void save(Game game) {
+    void save(Object game) {
         logger.debug "(Mock) Saving game: ${game}"
         store.put(game.id, game)
     }
 
     @Override
-    Game find(String gameId) {
+    Object find(String gameId) {
```

Comment:
We are not consistent here. Sometimes we use Object, sometimes we leave it empty. It is a super minor comment.